### PR TITLE
Update base images to ECR public

### PR DESCRIPTION
**Pull request by RedbackThomson**
Issue #, if available: https://github.com/aws-controllers-k8s/community/issues/925

Description of changes:
Update the base and builder image for the continuous deployment Dockerfiles

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: cd/scripts/release-controller.sh [hunk 1]
```diff
@@ -135,6 +135,9 @@ if ! (echo "$VERSION" | grep -Eq "stable$"); then
       echo " git commit: $SERVICE_CONTROLLER_GIT_COMMIT"
   fi
 
+  # Log into ECR public to access base images
+  aws ecr-public get-login-password --region us-east-1 | buildah login --username AWS --password-stdin public.ecr.aws
```
Comment (vijtrip2) [inline cd/scripts/release-controller.sh:139]: no need for this. line#111 already does the login.

Comment (RedbackThomson) [reply cd/scripts/release-controller.sh]: Oh. No idea how I missed that. Thanks!

**Review by vijtrip2 — COMMENTED** [review]
/lgtm

**Pull request merged**